elasticsearch/README.md

```diff
@@ -0,0 +1,51 @@
+# Elasticsearch
+
+## Image Utilities
+Following are utilities included in the image to facilitate running commonly used
+administration commands.  Use of these utilities assumes you have `exec` access
+to the container.  Some of these utilities make use of the administration certs which have
+full access to all indices and operations of the Elasticsearch cluster.
+
+### es_util
+Run any REST command against the Elasticsearch endpoint
+
+Example to retrieve the list of nodes
+```
+$ oc exec -c elasticsearch $POD -- es_util --query=_cat/nodes?pretty
+```
+
+### allocate-shard
+Manually allocate a shard to a given node in the Elasticsearch cluster.  The node
+by default will be the one on which the command is executed
```

Comment:
Trailing period ...

Perhaps we could say when one might want to use this command?  Or perhaps references the API docs in Elasticsearch?

Reply:
Adding link to https://www.elastic.co/guide/en/elasticsearch/reference/2.4/cluster-reroute.html